Utils/src/TriggerProducer.cc

```diff
@@ -150,9 +162,36 @@ TriggerProducer::produce(edm::StreamID, edm::Event& iEvent, const edm::EventSetu
     }
   }
 
+  edm::Handle<edm::TriggerResults> triggerBits;
+  iEvent.getByToken(trigResultsToken,triggerBits);
+
+  const edm::TriggerNames& names = iEvent.triggerNames(*triggerBits);
+  edm::Handle<pat::TriggerObjectStandAloneCollection> triggerObjects;
+  iEvent.getByToken(trigObjCollToken,triggerObjects);
+  //save the trigger object corresponding to the trigger HLT_Ele27_WPTight_Gsf_v*. Obtained code from https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookMiniAOD2016#Trigger
+  // loop over selected trigger objects                                                                                              
+  for (pat::TriggerObjectStandAlone obj : *triggerObjects) {
+    obj.unpackPathNames(names);
+    if(obj.pt() < 25.0) continue;//look for HLT objects with Pt > 25GeV only.
+    TLorentzVector objVec(obj.px(),obj.py(),obj.pz(),obj.energy());
+    std::vector<std::string> pathNamesAll = obj.pathNames(false);
+    std::vector<std::string> pathNamesLast = obj.pathNames(true);
+    for (unsigned h = 0, n = pathNamesAll.size(); h < n; ++h) {
+      bool isBoth = obj.hasPathName( pathNamesAll[h], true, true );//object is associated wih l3 filter and associated to the last filter of a successfull path. this object caused the trigger to fire.
+      std::string path_i = pathNamesAll[h];
+      path_i.pop_back(); path_i.pop_back(); //remove last 2 characters from path name.
```

Comment:
remove this and in the condition on the next line, just use: `path_i.find("HLT_Ele27_WPTight_Gsf_")!= std::string::npos`